csr_check.go

```diff
@@ -289,10 +299,20 @@ func recentlyPendingCSRs(indexer cache.Indexer) int {
 			continue
 		}
 
-		if !isApproved(csr) {
-			pending++
+		if isDenied(csr) {
+			denied++
 		}
 	}
 
-	return pending
+	return denied
+}
+
+func isNodeBootstrapCSR(csr *certificatesv1beta1.CertificateSigningRequest) bool {
```

Comment:
This function should return an error. `(bool, error)`. Let the caller decide what to do if there is an error.